docs/tutorial/yurt-e2e-test.md

```diff
@@ -0,0 +1,31 @@
+# Yurt-e2e-test 
+This tutorial shows how to build binary to \_output/bin/$GOOS/$GOARCH/yurt-e2e-test, such as: \_output/bin/darwin/amd64/yurt-e2e-test. And how to run yurt-e2e-test. We assume a minikube cluster  (version 1.14 or less) is installed and the cluser has been converted to Openyurt.
+
+## Build
+This tutorial demonstrates how to use `yurt-e2e-test` to test OpenYurt.
+1) First, enter openyurt work path:
+```bash
+ $cd openyurt
+```
+
+2) Then, build test binary:
+```bash
+$ make e2e
+```
+or as below:
+```
+$ make WHAT=test/e2e/yurt-e2e-test
+```
+
+## RUN Test
+1) So,you can use yurt-e2e-test binary to test openyurt.
```

Comment:
The functionality of running the test automatically is under development. For now, after the `yurt-e2e-test` has been successfully built,  we can run the test manually: